index.js

```diff
@@ -53,37 +53,48 @@ class KeepAChangelog extends Plugin {
   }
 
   getChangelog(latestVersion) {
-    const { isIncrement } = this.config;
-    // Return the unchanged changelog content when no increment is made
-    if (!isIncrement) {
-      return this.changelogContent;
-    }
-
     const { changelog } = this.getContext();
     if (changelog) return changelog;
 
     const { filename, strictLatest } = this;
-
     const previousReleaseTitle = strictLatest ? `## [${latestVersion}]` : `## [`;
     const hasPreviousReleaseSection = this.changelogContent.includes(previousReleaseTitle);
-
     if (strictLatest && !hasPreviousReleaseSection) {
       throw Error(`Missing section for previous release ("${latestVersion}") in ${filename}.`);
     }
 
-    const startIndex = this.changelogContent.indexOf(this.unreleasedTitle) + this.unreleasedTitle.length;
-    let endIndex = this.changelogContent.indexOf(previousReleaseTitle, startIndex);
-    if (!strictLatest && endIndex === -1) {
-      endIndex = this.changelogContent.length;
+    const { isIncrement } = this.config;
```

Comment:
This is where the new code starts. Everything between `if (changelog) return changelog;` and `const { isIncrement } = this.config;` is just to satisfy throwing an error if strictLatest is true and the changelog has never had a release before.